src/main/g8/src/main/scala/$package$/QuickstartApp.scala

```diff
@@ -0,0 +1,47 @@
+package $package$
+
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.adapter._
+import akka.actor.typed.scaladsl.Behaviors
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Route
+
+import scala.util.Failure
+import scala.util.Success
+
+//#main-class
+object QuickstartApp {
+
+  // #start-http-server
+  private def startHttpServer(routes: Route, system: ActorSystem[_]): Unit = {
+    // Akka HTTP still needs a classic ActorSystem to start
+    implicit val classicSystem: akka.actor.ActorSystem = system.toClassic
+    import system.executionContext
+
+    val futureBinding = Http().bindAndHandle(routes, "localhost", 8080)
+    futureBinding.onComplete {
+      case Success(binding) =>
+        val address = binding.localAddress
+        system.log.info("Server online at http://{}:{}/", address.getHostString, address.getPort)
+      case Failure(ex) =>
+        system.log.error("Failed to bind HTTP endpoint, terminating system", ex)
+        system.terminate()
+    }
+  }
+  // #start-http-server
+
+  def main(args: Array[String]): Unit = {
+    val rootBehavior = Behaviors.setup[Nothing] { context =>
+      val userRegistryActor = context.spawn(UserRegistry(), "UserRegistryActor")
+      context.watch(userRegistryActor)
```

Comment:
intended death pact here?

Reply:
Yep, if user registry stops the app won't work, so we want the actor system/entire app to stop